content/docs/integrations/roadie-api/index.md

```diff
@@ -0,0 +1,108 @@
+
+## Entity Sets
+
+To create / update a set of entities, you can call the following API. It will completely replace all entities in the specified set with the entities specified in the body of the request. If the set does not exist, it will be created as part of this call.
+
+```bash
+curl -X PUT https://api.roadie.so/api/catalog/roadie-entities/sets/my-entities \
+  -H "Authorization: Bearer $ROADIE_API_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+        "items": [
+          {
+             "apiVersion": "backstage.io/v1alpha1",
+             "kind": "Component",
+             "metadata": {
+               "name": "sample-component",
+               "description": "A sample component.",
+               "title": "Sample Component"
+             },
+             "spec": {
+               "type": "library",
+               "owner": "roadie-demo/dev-team",
+               "lifecycle": "production",
+               "system": "core-system"
+            }
+          }
+        ]
+      }'
+```
+
+You can list entity sets with the following API call.
+
+```bash
+curl -X GET https://api.roadie.so/api/catalog/roadie-entities/sets \
+  -H "Authorization: Bearer $ROADIE_API_TOKEN"
+```
+
+And list the entities within a set with the following call:
+
+```bash
+curl -X GET https://api.roadie.so/api/catalog/roadie-entities/entities?set=my-entities \
+  -H "Authorization: Bearer $ROADIE_API_TOKEN"
+```
+
+You can delete an entity set by removing the last entity from a particular set. e.g. the following call will delete the set:
+
+```bash
+curl -X PUT https://api.roadie.so/api/catalog/roadie-entities/sets/my-entities \
+  -H "Authorization: Bearer $ROADIE_API_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+        "items": []
+      }'
+```
+
+## Entities
+
+To create an entity without an entity set, you can do that with the following API call.
+
+```bash
+curl -X POST https://api.roadie.so/api/catalog/roadie-entities/entities \
+  -H "Authorization: Bearer $ROADIE_API_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+        "apiVersion": "backstage.io/v1alpha1",
+        "kind": "Component",
+        "metadata": {
+          "name": "another-sample-component",
+          "description": "Another sample component for testing Roadie Backstage functionality.",
+          "title": "Another Sample Component"
+        },
+        "spec": {
+          "type": "library",
+          "owner": "roadie-demo/dev-team",
+          "lifecycle": "experimental",
+          "system": "core"
+        }
+      }'
+```
+
+You can retrieve an entity with the entity id.
```

Comment:
How do I get this entity ID ? Is on the UI when I inspect my entity that I added to my catalog?